endpoints.js

```diff
@@ -0,0 +1,28 @@
+exports.apiEndPoints = {
+  "GET /api": {
+    description:
+      "serves up a json representation of all the available endpoints of the api",
+  },
+  "GET /api/topics": {
+    description: "serves an array of all topics",
+    queries: [],
+    exampleResponse: {
+      topics: [{ slug: "football", description: "Footie!" }],
+    },
+  },
+  "GET /api/articles": {
+    description: "serves an array of all topics",
+    queries: ["author", "topic", "sort_by", "order"],
+    exampleResponse: {
+      articles: [
+        {
+          title: "Seafood substitutions are increasing",
+          topic: "cooking",
+          author: "weegembump",
+          body: "Text from the article..",
+          created_at: 1527695953341,
+        },
+      ],
+    },
+  },
+};
```

Comment:
why have we moved the data into a .js file instead of keeping it in the .json? 
also you need to update the endpoint json file with all of your endpoints, following the same format as above :)